eks/cluster-loader/clusterloader2/cluster-loader.gotmpl

```diff
@@ -46,17 +46,34 @@ metadata:
 spec:
   template:
     spec:
+      initContainers:
+        - name: clusterloader2
+          image: {{ .Image }}
+          command: ["/clusterloader"]
+          args: {{ .TestArgs }}
+          volumeMounts:
+            - name: config
+              mountPath: /etc/config
+            - name: reports
+              mountPath: /var/reports
+              readOnly: false
       containers:
-      - name: clusterloader2
-        image: {{ .Image }}
-        command: ["/clusterloader"]
-        args: {{ .TestArgs }}
-        volumeMounts:
-        - name: config
-          mountPath: /etc/config
+        - name: uploadresults
+          image: amazon/aws-cli:latest
+          command: 
+            - /bin/sh
+            - -c
+            - >
```

Comment:
No need to do ">". That's for multiline strings. Your command is a one liner.
I'd recommend:
```
command:
  - /bin/sh
  - -c
arguments:
  -  aws s3 cp /var/reports/cluster-loader {{ .S3Uri }} --recursive
```
That will make everything but arguments very copy pastable.

Reply:
Got it.